.github/workflows/ci.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v1
         with:
-          go-version: 1.17
+          go-version: 1.19
       - run: GOPROXY=https://proxy.golang.org,direct GOSUMDB=off GO111MODULE=on go get -u github.com/kisielk/errcheck; /home/runner/go/bin/errcheck -tags draft ./...
```

Comment:
`go get -u` has been deprecated

See previous PR from `meshery-istio` for comparison

Reply:
I believe that I have addressed this now.